DOCUMENTATION.md

```diff
@@ -879,9 +883,37 @@ The concept that the VR user has a physical in game presence which is accomplish
   * **Headset Y Offset:** The box collider which is created for the user is set at a height from the user's headset position. If the collider is required to be lower to allow for room between the play area collider and the headset then this offset value will shorten the height of the generated box collider.
   * **Ignore Grabbed Collisions:** If this is checked then any items that are grabbed with the controller will not collide with the box collider and rigid body on the play area. This is very useful if the user is required to grab and wield objects because if the collider was active they would bounce off the play area collider.
   * **Reset Position On Collision:** If this is checked then if the Headset Collision Fade script is present and a headset collision occurs, the Camera Rig is moved back to the last good known standing position. This deals with any collision issues if a user stands up whilst moving through a crouched area as instead of them being able to clip into objects they are transported back to a position where they are able to stand.
+  * **Falling Physics Only**: Only use physics when an explicit falling state is set.
 
+### Class Events
+
+  * `PresenceFallStarted` - Emitted when a gravity based fall has started
+  * `PresenceFallEnded` - Emitted when a gravity based fall has ended
+  
 ### Class Methods
 
+#### SetFallingPhysicsOnlyParams/1
+
+  > `public void SetFallingPhysicsOnlyParams(bool falling)`
+
+  * Parameters
+   * `bool falling` - Toggle the physics falling on or off.
+  * Returns
+   * _none_
+
+The SetFallingPhysicsOnlyParams method will toggle the `fallingPhysicsOnly` class state as well as enable or disable physics if needed.
+
+#### IsFalling/0
+
+  > `public bool IsFalling()`
+
+  * Parameters
+   * _none_
+  * Returns
+   * `bool` - Returns if we are in a physics falling state or not.
```

Comment:
replace the word `we` for the words `the user`

so:

Returns if we are in a physics falling state or not.

should be:

Returns if **the user is** in a physics falling state or not.`